src/internal/auth/interceptor.go

```diff
@@ -250,10 +250,18 @@ func (i *Interceptor) InterceptUnary(ctx context.Context, req interface{}, info
 		return nil, fmt.Errorf("no auth function for %q, this is a bug", info.FullMethod)
 	}
 
-	if err := a(pachClient, info.FullMethod); err != nil {
+	authResp, authErr := a(pachClient, info.FullMethod)
+
+	if authErr != nil {
 		logrus.Errorf("denied unary call %q\n", info.FullMethod)
```

Comment:
If we get back the username we should include it in these logs (or "unauthenticated" if the user wasn't authed at all).